.github/workflows/test_araroot.yml

```diff
@@ -0,0 +1,68 @@
+name: test_araroot
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+
+      - name: Install dependencies
+        run: |
+          
+          sudo apt-get install dpkg-dev cmake g++ gcc binutils libx11-dev \
+           libxpm-dev libxft-dev libxext-dev python libssl-dev libfftw3-dev libgsl-dev libboost-dev
+
+
+      - name: Install ROOT
+        run: |
+          
+          wget -q https://root.cern/download/root_v6.22.06.Linux-ubuntu20-x86_64-gcc9.3.tar.gz
```

Comment:
check to make sure this is the right gcc? otherwise might lead to strange ABI issues.

Reply:
So, the test runs (e.g., AraRoot builds and runs), so I think it works? It's the only precompiled binary that ROOT offers for Ubuntu 20, so I think we're stuck with it anyway..